Formula/binutils.rb

```diff
@@ -12,13 +13,15 @@ class Binutils < Formula
     sha256 "7c693e68ebfcd654fd781480e68469e8f5af93c7397754a8caae1937322feb8f" => :el_capitan
   end
 
-  # No --default-names option as it interferes with Homebrew builds.
+  # See https://sourceware.org/bugzilla/show_bug.cgi?id=23728 and
+  # https://github.com/Homebrew/homebrew-core/issues/32516
+  keg_only :provided_by_macos,
+           "because Apple provides binutils and gnm is broken and breaks gcc"
```

Comment:
Either we decide that keg-only is temporary, and the comment (and links) is adequate, or we decide that even when gnm is fixed we'd prefer binutils to stay keg-only (which I think is preferable). But then discussion about gnm bugs is irrelevant.

Reply:
> decide that even when gnm is fixed we'd prefer binutils to stay keg-only (which I think is preferable)

👍 I'll remove comments.